docker/install/ubuntu_install_universal.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+set -u
+set -o pipefail
+
+git clone https://github.com/stillwater-sc/universal.git UNIVERSAL
```

Comment:
I suggest we use absolute paths (rather than relative `UNIVERSAL`), to know where this is going to land on the Docker image, independent on where it runs. Some scripts will put libraries and dependencies under `/opt/`.

This is the first time I hear about universal, please correct me if my comment doesn't apply, but also seems a bit unconventional to have an all caps directory name. I suggest we use regular lower case naming, perhaps `/opt/universal`?

Reply:
yes you are right, I'll move it under /opt/universal. thanks!